src/Microsoft.AspNet.Authentication.OpenIdConnect/DefaultIdentityModelLoggingListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    /// <summary>
+    /// Default logging event listener for wilson event source. It logs the data to the target where other Asp.Net logs go.
+    /// </summary>
+    public class DefaultIdentityModelLoggingListener : EventListener
```

Comment:
AspNet/Logging a library developed by the ASP.NET team that we are considering contributing back to the CLR team/repo. That will not be resolved any time soon, so it's fine not to use it.